src/object/srv_obj.c

```diff
@@ -674,17 +682,18 @@ obj_update_prefw(crt_rpc_t *req, void *arg)
 }
 
 static int
-ds_obj_rw_local_hdlr(crt_rpc_t *rpc, uint32_t tag, struct ds_cont_hdl *cont_hdl,
-		     struct ds_cont *cont, daos_handle_t *ioh, bool update,
+ds_obj_rw_local_hdlr(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
```

Comment:
it is a private function, obj_rw() should be a clear enough name? I prefer to use _hdlr as postfix only for real RPC handler.

Reply:
ok, I will change that